src/storage/storage_structure/column.cpp

```diff
@@ -189,14 +201,20 @@ void ListPropertyColumn::writeValueForSingleNodeIDPosition(
             ((ku_list_t*)(updatedPageInfoAndWALPageFrame.frame +
                           mapElementPosToByteOffset(updatedPageInfoAndWALPageFrame.posInPage)));
         auto kuListToWriteFrom = vectorToWriteFrom->getValue<ku_list_t>(posInVectorToWriteFrom);
-        diskOverflowFile.writeListOverflowAndUpdateOverflowPtr(
-            kuListToWriteFrom, *kuListToWriteTo, vectorToWriteFrom->dataType);
+        try {
```

Comment:
Can u write a comment explaining what's going on here in general and why you're releasing originalPageLock if there is an exception (when was the lock acquired (I believe in beginUpdatingPage but it's not very clear)).

Reply:
Done.